FinniversKit/Sources/Components/FrontPageTransactionView/FrontPageTransactionView.swift

```diff
@@ -0,0 +1,201 @@
+        label.textColor = .licorice
+        label.numberOfLines = 1
+        label.textAlignment = .left
+        return label
+    }()
+    
+    private lazy var subtitleLabel: UILabel = {
+        let label = Label(withAutoLayout: true)
+        label.font = .body
+        label.textColor = .licorice
+        label.numberOfLines = 1
+        label.textAlignment = .left
+        label.widthAnchor.constraint(equalToConstant: 228).isActive = true
+        return label
+    }()
+    
+    private lazy var imageView: RemoteImageView = {
+        let imageView = RemoteImageView(withAutoLayout: true)
+        imageView.image = UIImage(named: ImageAsset.noImage)
+        imageView.contentMode = .scaleAspectFill
+        imageView.layer.masksToBounds = true
+        imageView.widthAnchor.constraint(equalToConstant: imageWidth).isActive = true
+        imageView.heightAnchor.constraint(equalToConstant: imageWidth).isActive = true
+        imageView.layer.cornerRadius = cornerRadius
+        imageView.clipsToBounds = true
+        return imageView
+    }()
+    
+    private lazy var verticalStack: UIStackView = {
+        let stack = UIStackView(withAutoLayout: true)
+        stack.axis = .vertical
+        stack.alignment = .leading
+        stack.distribution = .fillProportionally
+        return stack
+    }()
+    
+    private lazy var backgroundContainer: UIView = {
+        let view = UIView(withAutoLayout: true)
+        view.backgroundColor = .white
+        view.layer.cornerRadius = cornerRadius
+        return view
+    }()
+    
+    private lazy var imageContainerView: UIView = {
+        let view = UIView(withAutoLayout: true)
+        view.backgroundColor = .bgColor
+        view.layer.cornerRadius = cornerRadius
+        view.layer.borderWidth = 1
+        view.layer.borderColor = UIColor.black.withAlphaComponent(0.1).cgColor
+        return view
+    }()
+    
+    private lazy var smallShadowView: UIView = {
+        let view = UIView(withAutoLayout: true)
+        view.dropShadow(
+            color: .shadowColor,
+            opacity: 0.24,
+            offset: CGSize(width: 0, height: 1),
+            radius: 1
+        )
+        return view
+    }()
+
+    private lazy var largeShadowView: UIView = {
+        let view = UIView(withAutoLayout: true)
+        view.dropShadow(
+            color: .shadowColor,
+            opacity: 0.16,
+            offset: CGSize(width: 0, height: 1),
+            radius: 5
+        )
+        return view
+    }()
+    
+    private lazy var horizontalStack: UIStackView = {
+        let stack = UIStackView(withAutoLayout: true)
+        stack.axis = .horizontal
+        stack.distribution = .fillProportionally
+        stack.spacing = .spacingXL
+        return stack
+    }()
+
+    private lazy var headerLabel: UILabel = {
+        let label = Label(withAutoLayout: true)
+        label.font = .bodyStrong
+        label.textColor = .textPrimary
+        return label
+    }()
```

Comment:
`.textPrimary` is already the default for `Label`, so this can be shorter.

```suggestion
    private lazy var headerLabel = Label(style: .bodyStrong, withAutoLayout: true)
```

Reply:
😄